app/Blocks/InsetText/build/block.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "name": "govukblogs/inset-text",
+  "version": "0.1.0",
+  "title": "GOV.UK Inset Text",
+  "category": "theme",
+  "icon": "smiley",
```

Comment:
We  can probably remove this, as icon set elsewhere

Reply:
Good spot, thanks.